js/oceanex.js

```diff
@@ -0,0 +1,323 @@
+            'amount': this.safeFloat (trade, 'volume'),
+            'cost': undefined,
+            'fee': undefined,
+        };
+    }
+
+    async fetchTime (params = {}) {
+        let response = await this.publicGetTimestamp ();
+        return response['data'];
+    }
+
+    async fetchTradingFees (params = {}) {
+        let response = await this.publicGetFeesTrading ();
+        return response['data'];
+    }
+
+    async fetchKey (params = {}) {
+        // private method input value {'user_jwt': encrypted_jwt_token}
+        // jwt_token should be RS256 encoded
+        let response = await this.privateGetKey (this.extend (params));
+        return response['data'];
+    }
+
+    async fetchBalance (params = {}) {
+        let response = await this.privateGetMembersMe (this.extend (params));
+        const balances = this.safeValue (response['data'], 'accounts');
+        let result = { 'info': balances };
+        for (let i = 0; i < balances.length; i++) {
+            const balance = balances[i];
+            let currency = balance['currency'];
+            let account = this.account ();
+            let free = this.safeFloat (balance, 'balance');
+            let used = this.safeFloat (balance, 'locked');
+            let total = free + used;
+            account['free'] = free;
+            account['used'] = used;
+            account['total'] = total;
+            result[currency] = account;
+        }
+        return this.parseBalance (result);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        let response = await this.privatePostOrders (this.extend (params));
+        const data = response['data'];
+        let result = {
+            'info': data,
+            'id': data['id'],
+            'symbol': data['market'],
+            'type': data['ord_type'],
+            'side': data['side'],
+            'status': data['state'],
+        };
+        return result;
+    }
+
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        let response = await this.privateGetOrders (this.extend (params));
+        const data = response['data'];
+        return this.parseOrders (data);
+    }
+
+    parseOrder (order, market = undefined) {
+        let result = {
+            'info': order,
+            'id': order['id'],
+            'timestamp': order['created_on'],
+            'datetime': order['created_at'],
+            'lastTradeTimestamp': undefined,
+            'symbol': order['market'],
+            'type': order['ord_type'],
+            'side': order['side'],
+            'price': this.safeFloat (order, 'price'),
+            'average': this.safeFloat (order, 'avg_price'),
+            'amount': this.safeFloat (order, 'volume'),
+            'remaining': this.safeFloat (order, 'remaining_volume'),
+            'filled': this.safeFloat (order, 'executed_volume'),
+            'status': order['state'],
```

Comment:
Also need to make that the order statuses are unified and parsed in a generic way with `this.parseOrderStatus`, check out an example here: https://github.com/ccxt/ccxt/blob/master/js/binance.js#L583

The list of unified order statuses as documented here: https://github.com/ccxt/ccxt/wiki/Manual#order-structure (`open`, `closed`, `canceled`).